clients/js/src/vaa.ts

```diff
@@ -90,15 +94,18 @@ export function parse(buffer: Buffer): VAA<Payload | Other> {
         .or(coreContractUpgradeParser)
         .or(portalContractUpgradeParser("TokenBridge"))
         .or(portalContractUpgradeParser("NFTBridge"))
+        .or(portalContractUpgradeParser("WormholeRelayer"))
         .or(portalRegisterChainParser("TokenBridge"))
         .or(portalRegisterChainParser("NFTBridge"))
+        .or(portalContractUpgradeParser("WormholeRelayer"))
```

Comment:
Shouldn't this be `.or(portalRegisterChainParser("WormholeRelayer"))`?

Reply:
Ah thank you for catching this - fixed!